editoast/src/models/stdcm_search_environment.rs

```diff
@@ -5,14 +5,47 @@ use diesel::ExpressionMethods;
 use diesel::QueryDsl;
 use diesel_async::RunQueryDsl;
 use editoast_derive::Model;
+use serde::Deserialize;
 use serde::Serialize;
+use std::collections::HashMap;
 use std::ops::DerefMut;
 use utoipa::ToSchema;
 
 use editoast_models::prelude::*;
 
-#[cfg(test)]
-use serde::Deserialize;
+#[editoast_derive::openapi_schema]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Default, ToSchema)]
+pub struct ArrayOfInt(Vec<i64>);
+
+impl ArrayOfInt {
```

Comment:
Now that we dropped the `Optional` part of this, can't we just drop entirely this type and refer directly to `Vec<i64>`. If the goal was to strongly type that it's a not just any `Vec` of `i64` (or any other reason that we need to keep it), then we can keep this type, but I would give it the name of what it is semantically, not technically: something like `OperationalPoints`?

Note: I see the type in DB is `Array<Nullable<Int8>>` and I wondered why we could not have a `Array<Int8>`. And looking a bit on the web (I'm no SQL expert 😅), it seems not trivial. And I'm guessing that might be another reason to keep this type `ArrayOfInt` around: to convert between `Vec<Option<i64>>` and `Vec<i64>`. Even in that case, I would rename the type to something less technical at the very least.